adsws/slackback/index.html

```diff
@@ -0,0 +1,58 @@
+<div class="modal fade" id="feedback-modal">
```

Comment:
I don't think the html belongs -- besides it won't work properly without javascript/css anyways